api3/global/openapi.gen.yaml

```diff
@@ -813,7 +811,6 @@ components:
       - starttime
       - status
       - tags
```

Comment:
what about tags? should it be `required`? in case of and error tags should be empty I think.